AutomatedTesting/Gem/PythonTests/scripting/TestSuite_Periodic.py

```diff
@@ -195,6 +195,7 @@ def test_Node_HappyPath_DuplicateNode(self, request, workspace, editor, launcher
         from . import Node_HappyPath_DuplicateNode as test_module
         self._run_test(request, workspace, editor, test_module)
 
+    @pytest.mark.xfail(reason="Test fails on nightly build builds, it needs to be fixed.")
```

Comment:
Recommend skipping this test instead of marking it XFail, that way we don't pay infrastructural costs for a test we're not paying attention to. Suggest tracking this with a GHI URL.

Reply:
Thank you for the recommendation, do you have the code and/or link to documentation on how to skip tests? I followed the pattern that was already in use within this same file which was the approach recommended.